packages/esm-ward-app/src/ward-patient-workspace/patient-history-card.component.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
What is this component for?  Is it related to this ticket?

Reply:
Oh yeah, stubbed that out before I realized it wasn't asked for in the ticket. Removing it.